### DISCUSSION TITLE
备用地址de.linuxgame.cn无法打开

### DISCUSSION THREAD

**Discussion by BakaCirno1** [Gamuxorg/bbs#311] [游戏或网站运行反馈]
可以ping通，但是丢包率100%
使用科学上网工具后亦是如此

**Comment by hyh19962008** [comment]
刷新几次就好了，不用FQ

Best

________________________________

**Comment by shouhuanxiaoji** [comment]
忘记服务器续费了。。已经好了

**Reply by Lanz-Han** [reply]
现在*404 no found*是什么问题

**Comment by liushengqi000** [comment]
我这两天是翻de.linuxgame.cn的google快照查路径，然后从kr.linuxgame.cn下载的，一把辛酸泪

**Comment by shouhuanxiaoji** [comment]
kr这个服务器是源服务器，速度快但是流量少，没有开放下载
